source/tmf-services/TMF638_Service_Inventory/config.json

```diff
@@ -0,0 +1,42 @@
+{
+
+
+  "db_prot": "mongodb",
+  "db_user": "mongodb",
+  "db_password": "mongodb",
+  "db_host": "canvas-svcinv-mongodb",
+  "db_port": 27017,
+  "db_name": "svcinv",
+
+  "alarm_host": "http://localhost:10011",
+  "alarm_url_hal": "/api/",
+
+  "redis_host": "redis",
+  "strict_schema": true,
+
+  "schemaValidationType": "USE_LOCAL_CACHE, USE_OPTIMISTIC_LOCAL, USE_LOCAL_MAPPING, USE_REMOTE_LOOKUP",
+
+  "localSchemaMapping": [
+    { "schema": "https://mycsp.com:8080/tmf-api/schema/Service/ServiceCandidate.yml", 
+      "file": "ServiceCandidate.yml" 
+    },
+    { "schema": "@schemaLocation", 
+      "file": "ServiceCandidate.yml" 
+    }
+  ],
+
+  "localSchemaLocation": "schema",
+
+  "injectListenerPostings": true,
+
+  "notExternal": true,
+  "servers" : [ { "url": "https://canvas-info.ihc-dt.cluster-3.de/tmf-api/serviceInventoryManagement/v5" } ], 
```

Comment:
this would bind it to cluster-3.de

Reply:
you are right. This was for my tests. It should be the internal address http://info.canvas.svc.cluster.local

Reply:
done